package.json

```diff
@@ -39,20 +41,23 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "lucify-component-builder": "~0.2.1",
+    "mocha": "^2.3.4",
     "simplify-geojson": "0.0.2",
-    "lucify-component-builder": "~0.2.1"
+    "babelify": "^6.4.0",
+    "console-browserify": "^1.1.0",
+    "gulp": "gulpjs/gulp#4.0",
+    "watchify": "^3.2.1",
+    "topojson": "^1.6.19",
+    "should": "^7.1.0"
   },
   "dependencies": {
     "area-polygon": "^1.0.1",
-    "babelify": "^6.4.0",
     "babyparse": "^0.4.3",
     "bluebird": "^2.10.0",
     "classnames": "^2.1.3",
-    "console-browserify": "^1.1.0",
```

Comment:
`console-browserify` actually is regular dependency despite its misleading name. It polyfills some console stuff for IE9